src/EnergyPlus/ZoneTempPredictorCorrector.cc

```diff
@@ -5014,113 +5014,67 @@ void DownInterpolate4HistoryValues(Real64 const OldTimeStep,
     //  is expected to be roughly integer-valued and near 2.0 or 3.0 or 4.0 or more.
 
     // first construct data on timestamps for interpolating with later
-    Real64 const oldTime0 = 0.0;
-    Real64 const oldTime1 = oldTime0 - OldTimeStep;
-
-    Real64 const newTime0 = 0.0;
-    Real64 const newTime1 = newTime0 - NewTimeStep;
-    Real64 const newTime2 = newTime1 - NewTimeStep;
-    Real64 const newTime3 = newTime2 - NewTimeStep;
-    Real64 const newTime4 = newTime3 - NewTimeStep;
-
     Real64 const DSRatio = OldTimeStep / NewTimeStep; // should pretty much be an integer value 2, 3, 4, etc.
 
     newVal0 = oldVal0;
 
     if (std::abs(DSRatio - 2.0) < 0.01) { // DSRatio = 2
         // first two points lie between oldVal0 and oldVal1
-        newVal1 = oldVal0 + (oldVal1 - oldVal0) * ((oldTime0 - newTime1) / (OldTimeStep));
-        newVal2 = oldVal0 + (oldVal1 - oldVal0) * ((oldTime0 - newTime2) / (OldTimeStep));
+        newVal1 = (oldVal0 + oldVal1) / 2;
```

Comment:
@rraustad Should these be `2.0` etc.? Or should this use `DSRatio` anyway here?

Reply:
I can be explicit here even though real/int = real. I'll critique this again when I make a new branch.